misc/build.func

```diff
@@ -129,6 +129,23 @@ ssh_check() {
   fi
 }
 
+generate_app_name() {
+    if ! command -v figlet &> /dev/null; then
+        sudo apt-get install -y figlet &> /dev/null
+    fi
+
+    # Use the Value of the APP Variable
+    APP="$1"
```

Comment:
```suggestion
```
We already have a global `APP` variable, so we don't need this input

Reply:
Not in this Moment. App Name will be declared later.

Reply:
Or, we would have to change all CT and move the APP name to position 1. But this makes the script confusing because the other variables are defined further down.

Reply:
Ah you are right